tests/integration/framework/process/injector/injector.go

```diff
@@ -54,7 +54,7 @@ func New(t *testing.T, fopts ...Option) *Injector {
 
 	fp := ports.Reserve(t, 3)
 	opts := options{
-		logLevel:      "info",
+		logLevel:      "debug",
```

Comment:
same here, is this getting reverted? Calling it out so we don't lose it in the large diff
```suggestion
		logLevel:      "info",
```

Reply:
Replied above :slightly_smiling_face: